src/mlpack/methods/ann/activation_functions/elish_function.hpp

```diff
@@ -0,0 +1,122 @@
+/**
+ * @file elish_function.hpp
+ * @author Bisakh Mondal
+ *
+ * Definition and implementation of the ELiSH function as described by
+ * Mina Basirat and Peter M. Roth.
+ *
+ * For more information see the following paper
+ * 
+ * @code
+ * @misc{ELiSHActivation,
```

Comment:
I think this should be Basirat2018. Let me know what you think.

Reply:
I appreciate your suggestion. Would love to change so. Btw isn't it just a name only required for adding citation ` \cite{}`  in LaTEX?

Reply:
Ok, so we Bibtex Shorthand which generally implies (as far as I know, not 100 % sure), that we write Author's last name and Year. Kindly refer [this](https://github.com/mlpack/mlpack/blob/928aee24c75f4227b5c1702aeb0acdc9aaf486e0/src/mlpack/methods/ann/layer/elu.hpp#L84). I haven't used shorthand much but if you have some reference that I could use to learn more that would be great. Thanks a lot.

Reply:
Yup bro you are actually correct, I have seen all previously used shorthand, they basically follow `author + pub. year + any special term(not necessary)` this format.
for eg: wgan-gp: `gulrajani2017improved`
GAN by goodfellow: `goodfellow2014generative`

Reply:
Great.